portfolio/src/main/webapp/index.html

```diff
@@ -5,17 +5,57 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+    a    {
+        color: black;
+        text-shadow: 2px 2px 5px turquoise, 1px 1px 3px purple, 1px 2px 3px snow;
+        text-decoration: underline;
+    }
+
+    body {
+        background: url(images/better.jpg) no-repeat center center fixed;
+    }
+
+    html {
+        scroll-behavior: smooth;
+    }
+
+    img  {
+        border-radius: 30px;
+    }
+
+    img:hover {
+        box-shadow: 0 0 2px 1px white;
+    }
+    </style>
   </head>
   <body>
     <div id="content">
-      <h1>Kyle Smith's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>I am a rising sophomore at MIT who is studying Mathematics & Computer Science.</p>
-      <p>I am an intern at Google</p>
-      <img src="/images/mit.jpg" width="300" height="177" alt="This is my university">
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Fact!</button>
-      <div id="fact-container"></div>
+      <div id="bold"><h1>Kyle Smith's Portfolio</h1></div>
+
+      <p><a href="https://github.com/kyleasmith1">GitHub</a>&nbsp;&nbsp;
+      <a href="about.html">About Me</a>&nbsp;&nbsp;
+      <a href="https://linkedin.com/in/k-smith23">LinkedIn</a></p>
+
+      <div id="bold"><h3>Click Here To See Some Cool Places I've Been To:</h3></div>
+      <a href="images.html">
+        <img src="/images/art.png" width="300" height="177" alt="Art Gallery">
+      </a>
+
+      <div id="bold"><h3>Some Projects I've Worked On:</h3></div>
```

Comment:
This is technically illegal HTML. Browsers are very forgiving and will try and figure out "what you probably meant" so it still may work, but to be well formed HTML, only one element in a document can have a given ID. The concept you are going for is a class. An ID says "This specific thing is unique and I need to be able to target this one exact element from CSS". A class says "This is in a category of things, all of which will be targeted by CSS"